### ISSUE TITLE
Renaming content types across the platform to make them shorter

### ISSUE THREAD

**Issue by acorbi** [OpenDevelopmentMekong/wp-odm_theme#1069]
As per @prustar's suggestion on https://github.com/OpenDevelopmentMekong/wp-odm_theme/issues/1032

In order to save up horizontal space on the search page, @prustar is proposing to rename the content types names as follows:

• # Agreements
• # Datasets -> Data
• # Laws
• # News articles -> News 
• # Profiles
• # Library Publications -> Publications
• # Topics

My suggestion is that, if we change this on the search page we should change it everywhere (menus, related content widgets, etc..)

@Huyeng @DBishton @gimmemochi @thetaung2 @DragonVirus . Would you agree about this change? Any suggestions/comments? 

Also, this relates to the intention we discussed of renaming Topics into Briefings. @DBishton If this is actually wished, I would address it as part of this task.

**Comment by gimmemochi** [comment]
@acorbi I agree with your abbreviation suggestions.

**Comment by rtritt** [comment]
@Huyeng @tthy @prustar @acorbi 
Try & Huy Eng - I also agree with this suggestion to rename the content in English as Pyrou suggests (better in English as it reduces redundancy, and makes more room).  I don't think this should impact the Khmer translation as the meaning in English is the same.  Please confirm if you agree to make this change in the serach page as well as everywhere else as Alex suggests.

**Comment by rtritt** [comment]
@Huyeng @tthy @acorbi @prustar @DBishton @gimmemochi 
In English, I also agree that "briefing" is more accurate than "topic" since in English the meaning of "topic" would actually be more applicable to what we now call "category".

**Comment by thetaung2** [comment]
@acorbi I agree with the suggestions. Agree to rename topics into briefings too.

**Comment by Huyeng** [comment]
Well, the News articles should be renamed to the News as the same on the menu. 

For renaming the datasets to the data, @tthy and the team thought that it a bit confuse to us. They will be a question what is different between the data and profile? That question was used to be asked before when the data page available on ODC site. Moreover, if these tow words are translated to Khmer Data "ទិន្នន័យ"  and Profile "ព័ត៌មានទិន្នន័យ", they are almost the same meaning that make the users be more confuse. That's way ODC doesn't have the data page on the site. Thus, our team would like to suggest to keep the dataset as the same. And renaming the datasets to the data doesn't save much space. 

In addition, ODM has Data page which contains Dataset, Agreement, Laws and Library while ODC splits them into different pages to make the users be easier to access what we have on the site.  
Reflect to this, if you rename Dataset to the Data on search tab, don't you think it will make the users be confused about this? On the navigation Data on ODM site covers the Dataset, Agreement, Laws and Library, but on the search page, the Data was separated from them. It means that you will have Agreement, Data, Laws, Library, and so on, thus it won't be make sense on this point. Will the Dataset, Agreement, Laws and Library be separated as ODC prefers on the search page, or will you replace them in with the word Data instead?

Talking about this data page, i would like to share a comment from the users who are the law students. They suggested that if the Dataset, Agreement, Laws and Library are placed under Data page, so there should be the drop down menu that list these items, otherwise, they don't know that the law can be found in the data page. 

It is fine for us to rename the "Library Publications" => "Publications".

Regarding to change the Topic to the Briefing, I would like to suggest to go back to review the history of changing the "Briefing" to "Topic",  and from the "Category" to the "Topic" before making the decision.

![screenshot_4](https://user-images.githubusercontent.com/6613530/27955795-ba874456-6340-11e7-95f6-e4e0b6a93f8b.jpg)

**Comment by acorbi** [comment]
Ok so far it seems that:

- We agree on the following changes

• # News articles -> News
• # Library Publications -> Publications

- There is still some arguments about

• # Datasets -> Data
* # Topics -> Briefings

I believe in case of changing Topics into Briefings, we should differentiate between:

- Topics as content type, as in "Topic pages" http://opendevelopmentmekong.net/topics/land. They should be renamed to briefings, as in Briefing pages, Land briefing page, etc... This would allow to make assertions such as: _ODM has published a new briefing page._

- Topics as way of categorize/filter content. On filter boxes (search page, data page, news archive page, etc...) Here we should keep the word topic as a more general concept which allow us to make assertions such as: _These news article is categorized under the "Land" topic_, _Users can filter search results by the main top-tier taxonomic topics_, etc...

**Comment by DBishton** [comment]
Hi all,

I agree with the suggestions put forward so far. @acorbi I also support your distinction here, it's important.  I support "Briefings" as the abbreviation for topics. 

>Topics as content type, as in "Topic pages" http://opendevelopmentmekong.net/topics/land. They should be renamed to briefings, as in Briefing pages, Land briefing page, etc... This would allow to make assertions such as: ODM has published a new briefing page.

I would just add that I prefer "topic briefings" to "briefing pages" as the 'extended' term for the content type.

**Comment by acorbi** [comment]
Thanks @DBishton for your comments:

I would then like to suggest the following to be implemented:

• Agreements
• Datasets 
• Laws
• News articles -> News
• Profiles
• Library Publications -> Publications
• Topics -> Briefings ( although the filter possibilities would still be named "topic", read the differentiation a made above between topics and briefings)
• Maps
• Site updates
• Announcements

**Comment by rtritt** [comment]
@Huyeng @acorbi 
Huy Eng referenced a previous discussion about the Topics/Briefings issue which i didn't see a link to. 
Huy Eng - what does ODC prefer?

**Comment by prustar** [comment]
@acorbi @Huyeng  -- 'DATA' vs 'Dataset' 

I think there are different ideas here which need to be clarified in terms of how we speak about the centralized CKAN instance that we all share, and is different from how data was managed previously by the legacy ODC site. If you hit the data page on ODM it does provide a list of resource types that are available in our database (CKAN): 

![image](https://user-images.githubusercontent.com/14996905/28227125-7b1f1ce2-688d-11e7-8d52-b40683f2e906.png)

These are represented as datasets (for numerical/geospatial data), Publications (which is basically a library), Laws and Agreements (which are also document repositories). 

In the case of profiles these are a specific collection of datasets that tell a collective story. 

I do agree that these terms might be particularly new to users and not clearly understood and distinguishable in local language. However in English they are clearer if distinguished this way and perhaps we should use the term Database (to represent CKAN) rather than just DATA as this is broader. My point is that in Khmer the term used should make sense in this language and all the other languages, but in English it should also make sense for users and ultimately we need to educate them on how we are organising the site and data. 

@rtritt @DBishton -- changing Topics to Briefings is here nor there to me, however it might be initially confusing for users until they get use to it.

**Comment by rtritt** [comment]
@prustar 
I'm a bit confused by the data-datasets discussion above.   In the current search results, it seems that datasets are actually referring to datasets - as there are also library publications, laws and agreements on the horizontal menu bar.
I assume the image you pasted above is from the Data button on our home page on the OD Mekong page.  Why does it say "search results for" when we are just clicking the data button?

**Comment by prustar** [comment]
@rtritt -- Thanks I was miss understanding the conversation - yes DATASETS

**Comment by Huyeng** [comment]
I just would like to clarify on renaming the topic to briefing before we make the decision. We would like to rename this in order to make it more sense and avoid confusing to the users or is there any purpose behind this? 

> Topics as content type, as in "Topic pages" http://opendevelopmentmekong.net/topics/land. They should be renamed to briefings, as in Briefing pages, Land briefing page, etc... This would allow to make assertions such as: ODM has published a new briefing page.

Since, the integrated platform has been deployed, the topic pages are creating following taxonomy. According to what @acorbi mentioned above, it seems that ODM would like to publish new pages of topic/briefing which are not available in the list of the topic/taxonomy? Please correct me if my understand is wrong.

I just would like to recall the history from what my understood, the old briefing pages of ODC have been updated and converted to the topics because we would like to create the interaction of the contents across the OD sites. And the topic pages of each country are required to has the same name based on the taxonomy. Right now, the interaction has been disable due to some challenges. Then, we would like to rename the topic to the briefing in order to freely produce the new pages which are not in the taxonomy. 

In the future, each country has many content as ODC, then the idea of interaction content will come up again, will the briefing page will change to the TOPIC page again?

**Comment by acorbi** [comment]
> We would like to rename this in order to make it more sense and avoid confusing to the users or is there any purpose behind this?

The reason for renaming **Topics** (as content type, not as filtering or classification information) into **Briefings** is to have a more appropriate/descriptive name for the content type. "Topic" is a broad concept, a piece of content (news articles, library publications, etc..) **might relate** to a topic and "Briefing" suits better for a content type which I would describe as a piece of editorial content providing information about a topic, which **does relate** to a topic.

> According to what @acorbi mentioned above, it seems that ODM would like to publish new pages of topic/briefing which are not available in the list of the topic/taxonomy? Please correct me if my understand is wrong.

This is not the case. By renaming "Topic" to "Briefing" (again, as content type) we would not change the editorial guidelines or the logic of the platform, it would only involve changing its name, nothing else.

> In the future, each country has many content as ODC, then the idea of interaction content will come up again, will the briefing page will change to the TOPIC page again?

Nope, this change should be maintained. I do not foresee any possible reason that would make use rename it again.

> For ODC, we are fine if just rename the Topic to Briefing, but others keep the as the same.

@Huyeng what do you mean by "others" in "but others keep the as the same."? please clarify

**Comment by Huyeng** [comment]
Well, I mean that it would like be fine for ODC for changing only the name, but nothing else related to logic of content publishing.

**Comment by acorbi** [comment]
OK, got it

**Comment by Huyeng** [comment]
If the topic will rename to the briefing, here are some tasks that need to do:
- Update the link of topic to briefing (maybe by script)
- Create the redirection from topic to briefing
- Update the texts/strings that mentioned about the topic page in some where such as the site update.
![screenshot_1](https://user-images.githubusercontent.com/6613530/28266376-b3e196fc-6b1e-11e7-9d8d-05096771dd7d.jpg)
https://opendevelopmentcambodia.net/updates

**Comment by acorbi** [comment]
@Huyeng 

> Update the link of topic to briefing (maybe by script)

You mean the URL? As in https://pp.opendevelopmentmekong.net/topics/ -> https://pp.opendevelopmentmekong.net/briefings or https://pp.opendevelopmentmekong.net/topics/land/ -> https://pp.opendevelopmentmekong.net/briefings/land/. If this is what you mean, yes, it will be done programmatically.

> Create the redirection from topic to briefing

We can do this with the **redirection** plugin

> Update the texts/strings that mentioned about the topic page in some where such as the site update.

This might be time-consuming for the editorial team. @DBishton can you help evaluating how many references to "topic page" are there on the regional and/or country sites?

**Comment by rtritt** [comment]
I assume Huy Eng is responding to the earlier discussion about changing the name on the top navigation bar and in search results from topics to briefings, since the category page is really the topic page that produces results on all contents. 
Based on Huy eng's  thoughtful comments I think this is more complex and we should wait to address this following a more strategic discussion.

**Comment by acorbi** [comment]
> Based on Huy eng's  thoughtful comments I think this is more complex and we should wait to address this following a more strategic discussion.

@rtritt I do not think this task is complex, at least not on the Tech side. In order to know the impact on editorial work, we would first need to estimate the editorial-related tasks.

**Comment by rtritt** [comment]
@Huyeng @acorbi I would follow Huy Eng and ODC's desire on this.  As Huy Eng has pointed out, renaming it from "topic page" to "briefing" on the front end could require a lot of editorial work since it is described as Topic Page on announcements, site updates, etc. Let's leave this for another discussion. It seems that Huy Eng and others clearly agree with the following changes for now.: 
• # News articles -> News
• # Library Publications -> Publications

**Comment by acorbi** [comment]
@Huyeng @rtritt OK, let's do the following. I have created another issue to address the renaming of the content types agreed sofar: https://github.com/OpenDevelopmentMekong/wp-odm_theme/issues/1109

For Topics -> Briefings, we continue the conversation and thinking process over here.
